### DISCUSSION TITLE
Merge commit messages

### DISCUSSION THREAD

**Discussion by gswifort** [CEXT-Dan/PyRx#21] [General]
Hi, I noticed that sometimes you provide incorrect Pull request numbers in commits when merging. This may be confusing for people who will be viewing the issues whose numbers you provided, e.g. commit [e122fbc](https://github.com/CEXT-Dan/PyRx/commit/e122fbc9fb805f2d921e1ba9e04e8468748014b0) refers to [Issues 1](https://github.com/CEXT-Dan/PyRx/issues/1#:~:text=CEXT%2DDan%20pushed%20a%20commit%20that%20referenced%20this%20issue%20last%20week)

**Comment by CEXT-Dan** [comment]
Oh, sorry about that! I don’t think I changed or added anything, only pressed the merge button
I’ll have to keep an eye on that. 

BTW, I haven’t forgot about issue 1. I’m still on the fence on how best to approach this.
